src/components/SegmentedControls/SegmentedControls.tsx

```diff
@@ -131,13 +133,17 @@ export const SegmentedControls = ({
         ));
     }, [items, id, disabled, radioGroupState]);
     const selectedIndex = items.findIndex((item) => item.id === radioGroupState.selectedValue);
+    const width = hugWidth ? '' : 'tw-w-full';
 
     return (
         <div className="tw-flex">
             <fieldset
                 {...radioGroupProps}
                 data-test-id="fondue-segmented-controls"
-                className="tw-relative tw-h-9 tw-w-full tw-grid tw-grid-flow-col tw-auto-cols-fr tw-justify-evenly tw-p-0 tw-border tw-border-black-20 tw-m-0 tw-bg-black-0 tw-rounded tw-font-sans tw-text-s tw-select-none"
+                className={merge([
```

Comment:
Thanks for handling, I think something more can be done here as it doesn't hug enough. We still have some extra space. It should hug `All` a bit more.
Design:
![image](https://github.com/Frontify/fondue/assets/35405438/5a95214b-06db-4a95-b3fc-da19e597437e)
When Inspecting:
![Screenshot 2023-05-15 at 16 54 03](https://github.com/Frontify/fondue/assets/35405438/535f24e3-b684-4842-9dd5-b14d38fda758)
We need to get rid of that purple space generated by flexbox when hugwidth is true.